06_create_cluster.sh

```diff
@@ -57,16 +57,15 @@ if [[ ! -z "${ENABLE_METALLB_MODE}" ]]; then
 		export METALLB_IMAGE_TAG="metallb"
 	fi
 
-	if [[ ${ENABLE_METALLB_MODE} == "l2" ]]; then
-		pushd metallb
-		./metallb_l2.sh
-		popd
-	elif [[ ${ENABLE_METALLB_MODE} == "bgp" ]]; then
+	pushd metallb
+	./configure_metallb.sh
+	popd
```

Comment:
I don't think metallb should be deployed unconditionally.  That will affect a bunch of CI jobs that we shouldn't mess with if we're not explicitly testing metallb.

Reply:
sorry, i missed that this was inside a conditional block on ENABLE_METALLB_MODE ... i blame the code being collapsed on github :-D